shoop/themes/classic_gray/static_src/js/custom.js

```diff
@@ -95,6 +95,10 @@ function updatePrice() {
     });
 }
 
+function changeLanguage() {
+    $("#language-field").attr("value", $(this).data("value")).closest("form").submit();
```

Comment:
Using `closest()` feels a little fragile. Add an ID to the form and use that?

EDIT: TBH, this shouldn't require a form in the DOM at all.

```javascript
const form = document.createElement("form");
form.method = "POST";
form.action = "/set-language/";
const input = document.createElement("input");
input.type = "hidden";
input.value = $(this).data("value");
form.appendChild(input);
document.body.appendChild(form);
form.submit();
```

 and so on, maybe?